src/main/java/cz/student/richard808code/App.java

```diff
@@ -1,4 +1,48 @@
 package cz.student.richard808code;
 
 public class App {
+    public static void main(String[] args) {
+        // Vytvoření uživatele
+
+        User user1 = new User();
```

Comment:
CLEAN-CODE: promenne pokud mozno tak nastavovat jako nemenne, tedy final v celem projektu, pokud dal promennou nebudes modifikovat tedy menit odkaz do pameti.

```suggestion
final User user1 = new User();
```

Reply:
Opraveno